sql/main_summary_v5.sql

```diff
@@ -0,0 +1,357 @@
+--   udf_get_theme(environment.addons.theme) AS active_theme,
+  environment.settings.blocklist_enabled,
+  environment.settings.addon_compatibility_check_enabled,
+  environment.settings.telemetry_enabled,
+
+  environment.settings.intl.accept_languages AS environment_settings_intl_accept_languages,
+  environment.settings.intl.app_locales AS environment_settings_intl_app_locales,
+  environment.settings.intl.available_locales AS environment_settings_intl_available_locales,
+  environment.settings.intl.regional_prefs_locales AS environment_settings_intl_regional_prefs_locales,
+  environment.settings.intl.requested_locales AS environment_settings_intl_requested_locales,
+  environment.settings.intl.system_locales AS environment_settings_intl_system_locales,
+
+  environment.system.gfx.headless AS environment_system_gfx_headless,
+
+  -- TODO: Deprecate and eventually remove this field, preferring the top-level user_pref_* fields for easy schema evolution.
+--   udf_get_old_user_prefs(environment.settings.user_prefs) AS user_prefs,
+
+--   udf_get_events([
+--     ("content", payload.processes.content.events),
+--     ("dynamic", payload.processes.dynamic.events),
+--     ("gfx", payload.processes.gfx.events),
+--     ("parent", payload.processes.parent.events)]) AS events,
+
+  -- bug 1339655
+--   udf_enum_histogram_bucket_count(payload.histograms.ssl_handshake_result, "0") AS ssl_handshake_result_success,
+--   udf_enum_histogram_sum_counts(payload.histograms.ssl_handshake_result,
+--     ARRAY(SELECT CAST(key AS STRING) FROM UNNEST(GENERATE_ARRAY(1, 671)) AS key)) AS ssl_handshake_result_failure,
+--   udf_enum_histogram_to_map(payload.histograms.ssl_handshake_result,
+--     ARRAY(SELECT CAST(key AS STRING) FROM UNNEST(GENERATE_ARRAY(0, 671)) AS key)) AS ssl_handshake_result,
+
+  -- bug 1353114 - payload.simpleMeasurements.*
+  COALESCE(
+--     udf_get_scalar_by_name(scalars, "scalar_parent_browser_engagement_active_ticks"),
+    payload.processes.parent.scalars.browser_engagement_active_ticks,
+    SAFE_CAST(JSON_EXTRACT_SCALAR(additional_properties, "$.payload.simple_measurements.active_ticks") AS INT64)) AS active_ticks,
+  SAFE_CAST(JSON_EXTRACT_SCALAR(additional_properties, "$.payload.simple_measurements.main") AS INT64) AS main,
+  COALESCE(
+--     udf_get_scalar_by_name(scalars, "scalar_parent_timestamps_first_paint"),
+    payload.processes.parent.scalars.timestamps_first_paint,
+    SAFE_CAST(JSON_EXTRACT_SCALAR(additional_properties, "$.payload.simple_measurements.first_paint") AS INT64)) AS first_paint,
+  SAFE_CAST(JSON_EXTRACT_SCALAR(additional_properties, "$.payload.simple_measurements.session_restored") AS INT64) AS session_restored,
+  SAFE_CAST(JSON_EXTRACT_SCALAR(additional_properties, "$.payload.simple_measurements.total_time") AS INT64) AS total_time,
+  SAFE_CAST(JSON_EXTRACT_SCALAR(additional_properties, "$.payload.simple_measurements.blank_window_shown") AS INT64) AS blank_window_shown,
```

Comment:
This is probably the best solution at the moment because a canonical list of simple measures doesn't exist.